vignettes/qgisprocess.Rmd

```diff
@@ -0,0 +1,271 @@
+---
+title: "Basic usage of qgisprocess"
+author: "Jannes Muenchow & Floris Vanderhaeghe"
+date: "Last updated: yyyy-mm-dd"
+output:
+  html_document: 
+    toc: yes
+    toc_float:
+      collapsed: no
+      smooth_scroll: no
+    toc_depth: 2
+vignette: >
+  %\VignetteIndexEntry{Basic usage of qgisprocess}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::knitr}
```

Comment:
```suggestion
  %\VignetteEngine{knitr::rmarkdown}
```

This appears to be the cause of most failing checks. Beware you also need to apply https://github.com/r-spatial/qgisprocess/pull/144#issuecomment-1483160720 for systems that lack QGIS (included in the checks).